core/scripts/start_site.py

```diff
@@ -0,0 +1,18 @@
+from ..app.ez_app import EZApplication
+from ..startup import ez_init
+
+
+def main():
+    ez_init.app = EZApplication()
+
+    import ez.lowlevel
+
+    ez.lowlevel.APP_HOST = ez_init.app.app_host
+    ez.lowlevel.EZ_APP = ez_init.app
+    ez.lowlevel.WEB_APP = ez_init.app.web_app
+
+    ez_init.app.run()
+
+    import ez.events as _
+    # import ez.log as _
```

Comment:
?

Reply:
So, in order to be able to access `events` from `ez` as if `ez` itself imported it, you need to import it at least once.

I added this because I want devs to be able to `import ez` and access all the APIs within it.

Reply:
Oh, the question mark is for the commented line.